src/components/Radio/index.stories.tsx

```diff
@@ -65,14 +65,14 @@ readingSelectedValue.parameters = {
   },
 };
 
-export const controlled = (args) => (
-  <div onChange={(e) => action('change')(e.target.value)}>
+export const controlled = (args: RadioProps): JSX.Element => (
+  <div onChange={(e) => action('change')((e.target as HTMLInputElement).value)}>
```

Comment:
no change required here, but I might create an issue to remove all the uses of this `action` method.

Reply:
https://github.com/cbinsights/form-design-system/issues/753